src/filesys.cpp

```diff
@@ -811,5 +813,15 @@ bool Rename(const std::string &from, const std::string &to)
 	return rename(from.c_str(), to.c_str()) == 0;
 }
 
+std::string TempFile()
+{
+	std::string path = TempPath() + DIR_DELIM "/MT_XXXXXX";
+	int fd = mkstemp(&path[0]); // modifies path
```

Comment:
are you sure mkstemp works on windows ?

Reply:
It is implemented on MinGW, someone else will have to fix MSVC if that's a problem.